main.go

```diff
@@ -1,5 +1,7 @@
 package main
 
+//go:generate esc -o server/assets.go -pkg server -modtime 1566640112 -ignore .DS_Store -prefix assets assets
```

Comment:
warum nach mein? das hat hier inhaltlich eigentlich nichts zu suchen?

Reply:
siehe zeile 3 in der pr/commit message